include/flashinfer/trtllm/common.h

```diff
@@ -226,6 +226,29 @@ enum Data_type {
   DATA_TYPE_UNKNOWN
 };
 
+inline constexpr const char* toStr(Data_type dtype) {
+  switch (dtype) {
+    case DATA_TYPE_FP16:
+      return "FP16";
+    case DATA_TYPE_BF16:
+      return "BF16";
+    case DATA_TYPE_FP32:
+      return "FP32";
+    case DATA_TYPE_INT8:
+      return "INT8";
+    case DATA_TYPE_INT32:
```

Comment:
We might also support `DATA_TYPE_INT64` as index data type in the future, but it's better to do that in another PR.